test/Api.Test/Controllers/GroupsControllerTests.cs

```diff
@@ -0,0 +1,143 @@
+﻿using Bit.Api.Controllers;
+using Bit.Api.Models.Request;
+using Bit.Core.Context;
+using Bit.Core.Entities;
+using Bit.Core.Exceptions;
+using Bit.Core.Models.Data;
+using Bit.Core.OrganizationFeatures.Groups.Interfaces;
+using Bit.Core.Repositories;
+using Bit.Test.Common.AutoFixture;
+using Bit.Test.Common.AutoFixture.Attributes;
+using NSubstitute;
+using Xunit;
+
+namespace Bit.Api.Test.Controllers;
+
+[ControllerCustomize(typeof(GroupsController))]
+[SutProviderCustomize]
+public class GroupsControllerTests
```

Comment:
FYI - I don't think we often unit test controllers, because they should just call other classes which have the important business logic in them (services/commands/queries). So we test those other classes instead. The controllers still have some logic in them, but they should be very simple, "flat" (non-nested) logic which can live without a unit test.

This is partly what I'm getting at in my other comments on this class - you end up doing quasi-integration tests, or indirectly testing dependencies, or testing not much at all. The happy path tests are fine, although I wouldn't require them, and I think we can remove the others. Just my 2 cents, tell me if you disagree.